Starship/Assets/Resources/Localization/Russian/Characters.xml

```diff
@@ -4,6 +4,9 @@
     <string name="Gender">Раса:</string>
     <string name="Experience">Опыт:</string>
     <string name="PlayerName">Это вы</string>
+    <string name="FirstNameHM">$[Джеймс|Давид|Кристофер|Джордж|Рональд|Джон|Ричард|Дэниэл|Кеннет|Энтони|Роберт|Чарльз|Пауль|Стивен|Кевин|Майкл|Джозеф|Марк|Эдвард|Джейсон|Уильям|Томас|Дональд|Брайан|Джефф]</string>
+    <string name="FirstNameHF">$[Мэри|Дженнифер|Лиза|Сандра|Мишель|Патриция|Мария|Нэнси|Донна|Лаура|Линда|Сьюзен|Карен|Кэрол|Сара|Барбара|Маргарет|Бетти|Руф|Кимберли|Элизабет|Дороти|Хелен|Шерон|Дебора]</string>
+    <string name="LastNameH">$[Смит|Джонсон|Уильямс|Джонс|Браун|Дэвис|Миллер|Уилсон|Мур|Тэйлор|Андерсон|Томас|Джексон|Уайт|Харрис|Мартин|Томсон|Гарсия|Мартинез|Робинсон|Кларк|Родригез|Льюис|Ли|Уолкер|Холл|Аллен|Янг|Фернандес|Кинг|Райт|Лопез|Хилл|Скотт|Грин|Адамс|Бейкер|Гонзалез|Нельсон|Картер|Митчелл|Перез|Робертс|Тернер|Филлипс|Кэмбелл|Паркер|Эванс|Эдвардс|Коллинс|Стьюарт|Санчез|Моррис|Роджерс|Рид|Кук|Морган|Белл|Мерфи|Бейли|Ривера|Купер|Ричардсон|Кох|Говард|Вард|Торрес|Питерсон|Грей|Рамирез|Джеймс|Ватсон|Брукс|Келли|Сандерс|Прайс|Беннет|Вуд|Барнс|Росс|Хендерсон|Коулман|Дженкинс|Перри|Пауелл|Лонг|Паттерсон|Хаджес|Флорес|Вашингтон|Батлер|Саймонс|Фостер|Гонсалес|Брайант|Александер|Рассел|Гриффин|Диас|Хейес]</string>
```

Comment:
I'm not sure if this is used anywhere but thanks, this can stay

Reply:
These are mostly EHF pilot names. It is questionable for me how EHF stuff will be implemented in EH.